libs/ptg-ui-web-components/src/components/ptg-tooltip/ptg-tooltip.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('ptg-modal', () => {
```

Comment:
@poojacsawai Update here from ptg-model to the tooltip component that we are using.